docker/resources/traefik-config.yml

```diff
@@ -0,0 +1,4 @@
+tls:
+  certificates:
+    - certFile: "/etc/certs/traefik.me.crt"
+      keyFile: "/etc/certs/traefik.me-key.pem"
```

Comment:
not necessary with `georchestra-127-0-0-1.traefik.me`

Reply:
I think you still need to download the files from http://traefik.me/